From 54bc31a7301f2e35e284eaa9c743089569b93f59 Mon Sep 17 00:00:00 2001
From: ChenXiaoSong <chenxiaosong@kylinos.cn>
Date: Sun, 7 Dec 2025 14:17:42 +0800
Subject: [PATCH 29/30] smb/client: remove useless elements from
 mapping_table_ERRSRV array

The last element in the array is no longer needed.

Signed-off-by: ChenXiaoSong <chenxiaosong@kylinos.cn>
---
 fs/smb/client/netmisc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/fs/smb/client/netmisc.c b/fs/smb/client/netmisc.c
index 8f3242c1d3da..8f93047305af 100644
--- a/fs/smb/client/netmisc.c
+++ b/fs/smb/client/netmisc.c
@@ -143,9 +143,7 @@ static struct smb_to_posix_error mapping_table_ERRSRV[] = {
 	{ERRbadclient, -EACCES},
 	{ERRbadLogonTime, -EACCES},
 	{ERRpasswordExpired, -EKEYEXPIRED},
-
 	{ERRnosupport, -EINVAL},
-	{0, 0}
 };
 
 static unsigned int errsrv_num = sizeof(mapping_table_ERRSRV) /
-- 
2.43.0

